frontend/public/components/pod.tsx

```diff
@@ -1021,13 +1021,20 @@ export const PodsPage: React.FC<PodPageProps> = ({
   const [data, filteredData, onFilterChange] = useListPageFilter(pods, filters, {
     name: { selected: [nameFilter] },
   });
-
+  const createNS = namespace || 'default';
+  const accessReview = {
+    groupVersionKind: referenceForModel(PodModel),
```

Comment:
Lets call `referenceForModel(PodModel)` only once and use that variable in both places.